manifests/params.pp

```diff
@@ -36,14 +36,15 @@
     $devel_package_name  = 'MariaDB-devel'
 
     # user.pp
-    $user    = 'mysql'
-    $comment = 'MySQL server'
-    $uid     = 494
-    $gid     = 494
-    $home    = '/var/lib/mysql'
-    $shell   = '/sbin/nologin'
-    $group   = 'mysql'
-    $groups  = undef
+    $user      = 'mysql'
+    $comment   = 'MySQL server'
+    $uid       = 494
+    $gid       = 494
+    $home      = '/var/lib/mysql'
+    $backupdir = '/var/lib/mysqlbackups'
```

Comment:
why did we choose this as the default backup dir?

Reply:
I found that if you put the backup dir in /var/lib/mysql.  It shows up as a database in show databases;    Doesn't seem like a good place to put it for that reason.